### PR TITLE
feat: add Catalyst

### PR DESCRIPTION
**Pull request by JaydenDev**
This pull request adds the Catalyst web browser into Scoop's main repository.

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/catalyst.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{ 
+    "version": "3.2.2",
+    "homepage": "https://getcatatalyst.eu.org",
+    "license": "MIT",
+    "notes": "As of the past few releases, 32-bit support has been unavailable due to the lack of Electron support for such architecture.",
+    "url": "https://github.com/JaydenDev/Catalyst/releases/download/v3.2.2/catalyst-3.2.2.Setup.exe",
+    "hash": "sha256:9d950fb4a4ad449fd8df4e8f12fc1b1d31e4d9fcaf93d3bb2bb1767b86636f26",
+    "bin": "catalyst-3.2.2.Setup.exe"
```
Comment (rasa) [inline bucket/catalyst.json:8]: That is a setup program, so this needs to be unpacked. See other manifests for examples.

**Review by rasa — CHANGES_REQUESTED** [review]

File: bucket/catalyst.json (hunk 1 above)
Comment (JaydenDev) [reply bucket/catalyst.json:8]: @rasa This executable was build with [Electron Forge](https://www.electronforge.io/) complicating the process, additional information is needed.

**Comment by rashil2000** [comment]
This package is too new for the official buckets. Please create a request for it in the Issues instead.

**Comment by JaydenDev** [comment]
> This package is too new for the official buckets. Please create a request for it in the Issues instead.

Too new? I'm confused, what in the package is too new? Is it the dependencies, Electron, the methods used?

**Comment by rashil2000** [comment]
You can refer to the criteria when creating a package request issue (it's present in the template)

**Pull request closed without merging**